src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -137,8 +137,9 @@ public boolean equals(Object other) {
         private Name name;
         private Phone phone;
         private Email email;
-        private Favourite favourite;
         private Address address;
+        private Favourite favourite;
+        private Set<Tag> tags;
```

Comment:
Is this change intentional? I don't think `Tag` exists anymore.

Reply:
Thanks for pointing out, will redo the changes!